analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -57,6 +57,16 @@ class Sbt(
         // https://github.com/sbt/sbt-launcher-package/blob/25c1b96/src/universal/bin/sbt#L449.
         private val BATCH_MODE = "-batch".takeUnless { Os.isWindows }.orEmpty()
 
+        // Enable the CI mode which disables console colors and supershell, see
```

Comment:
This sounds as if we could drop `LOG_NO_FORMAT` then, as `-Dsbt.ci=true` seems to do the same as `-Dsbt.log.noformat=true` (or `--color=false`) plus `-Dsbt.supershell=false` (or `--supershell=false`), right?

Reply:
True, I didn't realize that. I have updated the commit to replace the old option instead.